src/screens/SignTransactionSheet.tsx

```diff
@@ -505,17 +523,37 @@ export const SignTransactionSheet = () => {
       return;
     }
 
-    const existingWallet = await loadWallet(accountInfo.address, true, provider);
+    const existingWallet = await performanceTracking.getState().executeFn({
+      fn: loadWallet,
```

Comment:
wondering if there is a collision here as we'll be logging the "outer" `loadWallet` function as well as the "inner" `loadPrivateKey` function inside of `loadWallet` with the same screen / operation name that we're passing into `timeTracking` for `loadWallet`.

Reply:
Good catch! Should be Authentication for the inner one.